.github/CODEOWNERS

```diff
@@ -68,6 +68,9 @@ buf.yaml @anandwana001
 # Third-party dependencies.
 /third_party/ @BenHenning
 
+# License Texts XML.
+/app/src/main/res/values/third_party_dependencies.xml @BenHenning
```

Comment:
Does this need to go into the overrides section since app's XML files are blanket codeowner assigned below?

Reply:
Moved to overrides section